docs/essentials/gc/unmanaged/usingobjects.md

```diff
@@ -1 +1,153 @@
+# Using Objects That Implement IDisposable
+
+The common language runtime's garbage collector reclaims the memory used by unmanaged objects, but types that use unmanaged resources implement the [IDisposable](http://dotnet.github.io/api/System.IDisposable.html) interface to allow this unmanaged memory to be reclaimed. When you finish using an object that implements `IDisposable`, you should call the object's [IDisposable.Dispose](http://dotnet.github.io/api/System.IDisposable.html#System_IDisposable_Dispose) implementation. You can do this in one of two ways:
+
+* With the C# `using` statement.
+
+* By implementing a `try/finally` block. 
```

Comment:
I think the case where the disposable object is owned by another object should be mentioned here, with a link to *Implementing a Dispose Method*.